src/doc/gui_manual/Operators/OperatorTypes/ICS/LimitCycle_operator.rst

```diff
@@ -0,0 +1,110 @@
+.. _Limit Cycle operator:
+
+Limit Cycle operator
+~~~~~~~~~~~~~~~~~~~~
+
+The Limit Cycle Operator detects limit cycles within a vector field. Integral
```

Comment:
Subtle question...Are the limit cycles *in* the vector field or *in* the integral curves computed from the vector field?

Reply:
Limit cycle is an abstract mathematical construct. They are properties of a vector field, and one way to find them is by using integral curves.